src/scss/_patterns_icons.scss

```diff
@@ -112,7 +112,17 @@
 
   .p-icon--warning-grey {
     @extend %icon;
-    @include vf-icon-warning-grey($color-white)
+    @include vf-icon-warning-grey($color-white);
+  }
+
+  .p-icon--units-light {
```

Comment:
You don't need to declare a custom className for a light variant of each icon.

You can add the custom icon by importing it in the main .scss file (`@include vf-p-icon-machines;`) and then use `.p-icon-machines` in combination with `.is-light` 

This will be true for other icons in this file.

Reply:
@ndv99 I'll merge this as is, please investigate as part of https://warthogs.atlassian.net/browse/MAASENG-1342